README.md

```diff
@@ -1 +1,17 @@
-# -Hello-Microverse-
\ No newline at end of file
+# Hello-Microverse
```

Comment:
- [ ] [OPTIONAL] You should use a descriptive PR title that reflects the changes you made in the PR, kindly implement this